docker-build-package

```diff
@@ -104,7 +105,9 @@ output_dir="output/opensource/${DISTRO}-${RELEASE}-${ARCH}"
 if [ $commercial = "true" ]; then
     output_dir="output/commercial/${DISTRO}-${RELEASE}-${ARCH}"
 fi
-mkdir -p ${output_dir}
+mkdir -p "${output_dir}"
+find output -name '*.orig.tar.*'
+[[ "${SAVE_ORIG}" != "true" && "${DISTRO}-${RELEASE}-${ARCH}" != "debian-buster-amd64" ]] && cp -v output/opensource/debian-buster-amd64/*orig.tar.* "${output_dir}/"
```

Comment:
This hardcoded path is quite ugly. Suggestion: Copy the orig file to some common place instead, while building it, and then pick that up here instead of a hardcoded platform path.

Reply:
do you like [output/opensource/orig](https://github.com/mendersoftware/mender-dist-packages/pull/254/commits/1de010f32915945c1311780818ae0f2c1e01185f#diff-a696dd766c3d4eb6a58ba77a8c75d4f3888cf19cb5d7332f12a6214f07f0d5d1R108) better?